auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -141,12 +141,16 @@
           <span>{{ status(item) }}</span>
           <!-- Future Effective Icon for PAID_FE status -->
           <template v-if="status(item) === BusinessState.PAID_FE">
-            <v-tooltip bottom>
+            <v-tooltip
+              top
+              :max-width="'300px'"
```

Comment:
Try removing the binding indicator (:) and see if you can get rid of the internal quotes.

Reply:
Updated to use the IconTooltip sub-component